gcp-simulator/run.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+export DEV_URL='http://localhost:8080'
+
```

Comment:
export GCP_METADATA_URL = 'http://localhost:8080/metadata.google.internal/computeMetadata/v1'
export GCP_CLOUD_RESOURCE_MANAGER_URL = 'http://localhost:8080/cloudresourcemanager.googleapis.com/v1'
export GCP_SERVICE_USAGE_URL = 'http://localhost:8080/serviceusage.googleapis.com/v1'
export GCP_MONITORING_URL = 'http://localhost:8080/monitoring.googleapis.com/v3'

Can we do this instead?

Reply:
Sure, makes more sense!